packages/react-composites/playwright.config.common.ts

```diff
@@ -10,7 +10,11 @@ const DESKTOP_VIEWPORT = {
 };
 
 const TEST_ROOT = './tests/browser';
-const OUTPUT_DIR = './test-results';
+
+const outputDir = process.env.PLAYWRIGHT_OUTPUT_DIR;
+if (!outputDir) {
```

Comment:
Can this be a default if I don't want to set this for local runs?

Reply:
Nope. All runs should go via `runBrowserTests.mjs` (there is so much specific configuration in our repo....)